extensions/wikia/GameGuides/GameGuides_setup.php

```diff
@@ -42,16 +38,7 @@
 //$wgAutoloadClasses[ 'GameGuidesSpecialContentController'] =  "{$dir}/GameGuidesSpecialContentController.class.php" ;
 //$wgSpecialPages[ 'GameGuidesContent' ] =  'GameGuidesSpecialContentController';
 
-$wgGroupPermissions['*']['gameguidescontent'] = false;
-$wgGroupPermissions['staff']['gameguidescontent'] = true;
-$wgGroupPermissions['helper']['gameguidescontent'] = true;
-
-if ( $wgGameGuidesContentForAdmins ) {
-	$wgGroupPermissions['sysop']['gameguidescontent'] = true;
```

Comment:
I moved these lines as is to the new place, so the if will stay as is. You're right however that I can double check when is the wgGameGuidesContentForAdmins set to make sure we don't load rights before the value of this variable is set.